work_in_progress/agnostic_tools.ipynb

```diff
@@ -269,30 +282,9 @@
    "outputs": [],
    "source": [
     "%%time\n",
-    "metric = \"Average Max Air Temperature\"\n",
-    "single_stats_area, multiple_stats_area, results_area = agg_area_subset_sims('states', 'CA', metric, years, method, months)\n"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "21935e3b-cbe2-401d-b18a-f302a01f3718",
-   "metadata": {},
-   "source": [
-    "If we're interested in plotting two metrics aggregations each other for state-wide aggregation, we can run the following cell on a different metric to see how models quantitatively differ from one another across two variables.\n",
-    "\n",
-    "If you're not interested in a second variable, you can just skip the following cell."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "bb30abff-dfbd-4f99-8c20-63975a037b0e",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "%%time\n",
-    "metric = \"Average Annual Total Precipitation\"\n",
-    "single_stats_area2, multiple_stats_area2, results_area2 = agg_area_subset_sims('states', 'CA', metric, years, method, months)"
+    "area_subset = 'states' # Choose your `area_subset`\n",
+    "selected_area = 'CA' # Choose your `selected_area`\n",
```

Comment:
At this point, in order for the users to know which areas are available to select on, they would need some version of the `Select` object available to them. I think for demo purposes, we should leave this as is, but I think this again strongly suggests that the `Select` object be used here in some detail in order to validate arguments and showcase options to the user.